easybuild/easyconfigs/s/sep/sep-1.0.3-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'sep'
+version = '1.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://sep.readthedocs.io/en/v1.0.x/'
+description = """Python and C library for Source Extraction and Photometry.
+( this easyconfig provides python library only )"""
```

Comment:
style nitpicking: start continuation line with a space (and no space after '(' and before ')')